config/jetpack-cloud-development.json

```diff
@@ -61,6 +61,7 @@
 	"enable_all_sections": false,
 	"sections": {
 		"activity": true,
+		"auth": true,
```

Comment:
Enable the OAuth module/section on Jetpack cloud so users can access `/oauth-login`.